src/Language/PureScript/Publish/Registry/Compat.hs

```diff
@@ -53,21 +53,27 @@ data PursJson = PursJson
   , pursJsonLocation :: Text
     -- | An optional description of the package
   , pursJsonDescription :: Maybe Text
-    -- | The registry-compliant version of the package, which is SemVer minus
-    -- | build metadata and prerelease identifiers. This version can be used
-    -- | to associate this manifest with its metadata.
-  , pursJsonVersion :: Text
     -- | A map of dependencies, where keys are package names and values are
     -- | dependency ranges of the form '>=X.Y.Z <X.Y.Z'
   , pursJsonDependencies :: Map Text Text
   }
 
-asPursJson :: Parse Bower.BowerError PursJson
+data PursJsonError
+  = MalformedLocationField
+  deriving (Eq, Show, Ord, Generic)
+
+instance NFData PursJsonError
+
+showPursJsonError :: PursJsonError -> Text
+showPursJsonError = \case
+  MalformedLocationField ->
+    "The 'location' field must be either '{ \"githubOwner\": OWNER, \"githubRepo\": REPO }' or '{ \"gitUrl\": URL }'."
```

Comment:
How do you feel about this, @JordanMartinez?

Reply:
Works for me! Thanks!